src/ProjectDataBase/Tables/User.sql

```diff
@@ -1,11 +1,11 @@
 ﻿CREATE TABLE [dbo].[User]
 (
-    [Id] INT NOT NULL PRIMARY KEY IDENTITY,
-    [Login] TEXT NOT NULL,
-    [Password] TEXT NOT NULL,
-    [FirstName] TEXT NOT NULL,
-    [LastName] TEXT NOT NULL,
-    [Email] TEXT NOT NULL,
+    [Id] INT NOT NULL PRIMARY KEY IDENTITY(1,1),
+    [PasswordHash] NVARCHAR(300) NOT NULL,
```

Comment:
Are you sure that 300 symbols is enough? Have you checked hash and salt length which you are going to use?